cmake/compilers/X3D_flags_cray.cmake

```diff
@@ -0,0 +1,5 @@
+#Compilers Flags for Cray
+
+set(X3D_FFLAGS "-eF -g -N 1023")
```

Comment:
What is the meaning of 1023 here ?

Reply:
It's the line length that the compiler will accept.